### ISSUE TITLE
various for PMID:30393157

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3390]
NTR:
increased level of carbohydrate metabolism gene mRNA during  vegetative growth

Standard def
PMID:PMID:30393157

also consider:
i) adding "PS" synonyms to all terms referring to phophotidylinositol
ii) add synonym "decreased mitochondrial membrane depolarization" to "decreased mitochondrial membrane potential" etc 
iii) we have terms for "iron assimilation genes" and "iron starvation genes"
is there a difference?

**Comment by mah11** [comment]
> adding "PS" synonyms to all terms referring to phophotidylinositol

Absolutely not. PS is an abbreviation for phosphatidyl*serine*. All but one of the phosphatidylserine terms have synonyms with PS, and I'll add the missing one.

> add synonym "decreased mitochondrial membrane depolarization" to "decreased mitochondrial membrane potential" etc

This is backwards - a more polarized membrane (less depolarized) has a higher membrane potential. I can add synonyms the right way round. I wonder if the "depolarization" synonyms should be related, not exact ...

> we have terms for "iron assimilation genes" and "iron starvation genes"
> is there a difference?

I don't see the string "iron starvation genes" anywhere in FYPO. For glucose and nitrogen, there are terms that use "starvation genes", but for those responding to starvation presumably involves more than assimilating substances.

**Comment by ValWood** [comment]
> All but one of the phosphatidylserine terms have synonyms with PS, and I'll add the missing one.

yes that's what I meant sorry  FYPO:0005823

> This is backwards -

so can I annotation "decreased membrane potential" to 
FYPO:0006228 | increased plasma membrane potential
Do you think that's OK?

> I don't see the string "iron starvation genes" 
That's weird. I'm sure I saw them on the gene page. Maybe I was looking at GO annotation though

**Comment by mah11** [comment]
> so can I annotat[e] "decreased membrane potential" to
> FYPO:0006228 | increased plasma membrane potential
> Do you think that's OK?

No. This paper looks at the mitochondrial membrane potential, not plasma membrane.

And the thing that was backwards was equating decreased potential with decreased DEpolarization. It's closer to:

decreased membrane potential ~= decreased membrane polarization ~= increased membrane depolarization
and
increased membrane potential = increased membrane polarization ~= decreased membrane depolarization*

i.e. a more polarized membrane has a higher potential. Depolarization is a bit more complicated (yay biology) because it's not a thing the mitochondrial membrane normally does, unlike neurons. Depolarization can be induced by substances or mutations, but because it's not normal for mitochondria, I wouldn't add the decreased depolarization synonym I marked with an asterisk unless there was a lot of demand, and then not as an exact synonym. Figure 4 shows that trz2 overexpression causes loss of mitochondrial membrane potential, aka an abnormal occurrence of depolarization.

tl;dr: use FYPO:0004944 ! decreased mitochondrial membrane potential for trz2 overexpression.

**Comment by ValWood** [comment]
yes sorry, I was thinking mitochondrial membrane. I don't know why I selected plasma membrane...it was late...
sorted

**Comment by mah11** [comment]
OK, new term is
increased level of carbohydrate metabolism gene mRNA during vegetative growth FYPO:0006783

and a bunch of synonyms added to existing terms

edit file: 423aefa579afe58cec1e1c5f82fbec832dd7a73c
release: 52a117edfe20236d0d06a03446966bf7b440b67f
